hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -91,6 +92,57 @@
 	String lastUsedExportDir = UNKNOWN_EXPORT_DIR;
 	Instant periodEnd = NEVER;
 
+
+	private static final Comparator<AccountID> ACCOUNT_ID_COMPARATOR = Comparator
```

Comment:
There are accountID comparator and tokenId comparator which are public in HederaLedger. They can be reused.

Reply:
Right. Changed to import the comparators from HederaLedger class.